sections/errorhandling/returningpromises.md

```diff
@@ -0,0 +1,94 @@
+# Returning promises
+
+<br/>
+
+### One Paragraph Explainer
+
+The mechanisms behind sync function stacktrace and async function stacktrace in v8 implementation are quite different:
+sync stacktrace is based on **stack** provided by operation system Node.js is running on (just like in most programming
+languages). When async function is executing, the **stack** of operation system is popping it out as soon as the
```

Comment:
Here I think we could write either `When an async function is executing` or `When async function are executing` (but this one needs little changes in the rest of the text).